code/headers/st7735.hpp

```diff
@@ -116,7 +206,22 @@ namespace r2d2::display {
          * @param y_max
          */
         void set_cursor(uint16_t x_min, uint16_t y_min, uint16_t x_max,
-                        uint16_t y_max);
+                        uint16_t y_max) {
+            x_min += x_offset;
+            x_max += x_offset;
+            y_min += y_offset;
+            y_max += y_offset;
+
+            // set the min and max row
+            write_command(_CASET);
+            write_data(uint8_t(x_min >> 8), uint8_t(x_min & 0xFF),
```

Comment:
This has already be mentioned in the code review and at that point it was okey why is it not okay now?

Reply:
What do you mean? I don't know what code review you are referring to.

Reply:
the static cast part

Reply:
That was referring to the enumeration values in the initialization sequence for a display. The reason it was allowed there:
 - The enumeration is tagged with the `uint8_t` type
 - There was no immediate value in C++ style casts because the values aren't changing at runtime.

Both don't apply here, so please use C++ style casts.

Reply:
fixed in https://github.com/R2D2-2019/display/pull/36/commits/66eb88d6f24bbd2214d229054efb2d0856f1dc56